### PR TITLE
auto-detect InputView's returnKeyType, assign appropriate return key …

### PR DESCRIPTION
**Pull request by jk234ert**
…title and color

### CODE REVIEW

**Comment by MainasuK** [comment]
It looks great. Let me add some i18n.

**Comment by MainasuK** [comment]
Keyboard info.plist PrimaryLanguage is en-US so i18n not works. That's fine.

And two issue when switch between different return key type text field.
- The background color not always set success when it shouold be set.
- The key cap text color should be white color over blue background.

File: TesserCubeKeyboard/Controller/KeyboardLayout.swift [hunk 1]
```diff
@@ -514,6 +517,34 @@ class KeyboardLayout: NSObject, KeyboardKeyProtocol {
         }
     }
     
+    func updateReturnKeyType(_ returnType: UIReturnKeyType) {
+        for (key, view) in self.modelToView {
+            if key.type == .return {
+                key.uppercaseKeyCap = returnType.title
```
Comment (MainasuK) [inline TesserCubeKeyboard/Controller/KeyboardLayout.swift]: Maybe missing lowercaseKeyCap

Comment (jk234ert) [reply TesserCubeKeyboard/Controller/KeyboardLayout.swift]: This is some kind of trick for "special keys" like the return key.
Only setting uppercaseKeyCap and leave other _KeyCaps_ nil makes KeyboardLayout ignore other _KeyCaps_

For detail check:[HERE](https://github.com/DimensionDev/Tessercube-iOS/pull/73/files#diff-b1aa9bd594576b53e784800c81ff3d9dR157-R164)

**Comment by jk234ert** [comment]
@MainasuK have a test on the new commits

**Pull request merged**